api/src/LinkyLink/Controllers/LinksController.cs

```diff
@@ -27,14 +27,18 @@ public LinksController(ILinksService linksService)
             _linksService = linksService;
         }
 
-        // Todo - Remove this method
-        // GET: api/Links
```

Comment:
Can you delete this method.? Its not needed, I added that for testing, thats why there is a TODO to delete it later. I know its not in scope of this PR so skip it and I`ll delete it later.

Reply:
I can do it when I make changes to ValidationPage Controller.